app/src/main/java/com/odysee/app/ui/findcontent/FileViewFragment.java

```diff
@@ -125,16 +127,7 @@
 import java.util.List;
 import java.util.Map;
 import java.util.Objects;
-import java.util.concurrent.Callable;
-import java.util.concurrent.CompletableFuture;
-import java.util.concurrent.ExecutionException;
-import java.util.concurrent.ExecutorService;
-import java.util.concurrent.Executors;
-import java.util.concurrent.Future;
-import java.util.concurrent.ScheduledExecutorService;
-import java.util.concurrent.ScheduledFuture;
-import java.util.concurrent.ScheduledThreadPoolExecutor;
-import java.util.concurrent.TimeUnit;
+import java.util.concurrent.*;
```

Comment:
Please use individual imports, instead of wildcards. Everything else looks good.

Once this is fixed, I will merge. Thanks.

Reply:
Ok, done.